### ISSUE TITLE
Incorrect output from generateFontSource()

### ISSUE THREAD

**Issue by layershifter** [rofrischmann/fela#709]
**Type**: Bug

Pass `localAlias` option to `renderFont` and you will get weird output:

```css
@font-face {
  font-weight: 400;
  src: {src} /* <---- ???????? */ local('Segoe WP Light'), url('...') format('woff2');
  font-family:"Segoe UI"
}
```

| Packages | Version |
| -------- | ------- |
| fela     | 10.4.1   |

#### Description

Issue comes from broken template string in `generateFontSource()`:
```diff
-return `{src} local(${localUrl}), `
+return `${src} local(${localUrl}), `
```

**Comment by rofrischmann** [comment]
Huh, when was this once introduced :o 
Will fix asap!

**Comment by layershifter** [comment]
Actually I started to work on it 🐎 But, feel free to make PR 🐱 

There is one more issue: `files` and `fontLocals` are optional params, so I can pass empty array of `files` (it's our usecase).

After this fix it will receive:

```css
@font-face {
  font-weight: 400;
  src: local('Segoe WP Light'), ; /* <-- lonely comma 🐗  */
  font-family:"Segoe UI"
}
```

**Comment by rofrischmann** [comment]
Oh if you wanna fix it, go for it :) 
I never really touched the font loading for 2 years now.

**Comment by layershifter** [comment]
Fix in #710 🚀
